### PR TITLE
Convert citations (lines starting with ">") to HTML <blockquote>

### PR DESCRIPTION
**Pull request by Chris00**
Fixes https://github.com/jeremy-compostella/org-msg/issues/94

Signed-off-by: Christophe Troestler <Christophe.Troestler@umons.ac.be>

### CODE REVIEW

File: org-msg.el [hunk 1]
```diff
@@ -740,7 +757,8 @@ absolute paths."
 	    (org-html-head-include-scripts nil)
 	    (org-html-head-include-default-style nil)
 	    (org-msg-export-in-progress t))
-	(with-current-buffer (org-html-export-as-html)
+	(with-current-buffer
+	    (org-export-to-buffer 'org-msg-html "*Org Msg HTML Export*")
```
[the remark this answers is not in the document]
Comment (jeremy-compostella) [reply org-msg.el:761]: I am not a big fan of having a named temporary buffer. I would think a more anonymous buffer name would be more appropriate.  I tried to find a better construction but I could not come up with anything good.  Note that using `(current-buffer)` seems to work but the `org-export-to-buffer` documentation does not provide any guarantee that write in the same buffer is supported.

File: org-msg.el [hunk 2]
```diff
@@ -541,7 +542,7 @@ See `org-msg-css-to-list'."
 	(cl-find 'org-quote face)
       (eq 'org-quote face))))
 
-(defun org-msg-ascii-blockquote (&optional level begin end)
```
[the remark this answers is not in the document]
Comment (Chris00) [reply org-msg.el:544]: OK.

File: org-msg.el (hunk 1 above)
[the remark this answers is not in the document]
Comment (Chris00) [reply org-msg.el]: I tried with `(with-temp-buffer ... (buffer-name))` but it did not work.  I made a generated a “hidden buffer” (starting with a space).  This is basically what `with-temp-buffer` does.

**Comment by jeremy-compostella** [comment]
I  tested this pull request and it breaking a feature: each level should have a different color.

File: org-msg.el [hunk 3]
```diff
@@ -758,10 +775,12 @@ absolute paths."
 	    (org-html-head-include-scripts nil)
 	    (org-html-head-include-default-style nil)
 	    (org-msg-export-in-progress t))
-	(with-current-buffer (org-html-export-as-html)
-	  (let ((xml (org-msg-html-buffer-to-xml base)))
-	    (kill-buffer)
-	    xml))))))
+	(let ((buf (generate-new-buffer-name " *OrgMsg HTML Export*")))
+	  (with-current-buffer
+	      (org-export-to-buffer 'org-msg-html buf)
```
Comment (jeremy-compostella) [inline org-msg.el:780]: This should be on one line.

**Comment by Chris00** [comment]
> each level should have a different color.

Did you update `org-msg-default-style` (despite the fact that it is declared as a constant)?  This is needed for the appropriate styles to be applied.

**Comment by jeremy-compostella** [comment]
Indeed, I forgot that `eval-buffer`does not re-evaluate the value of `defcustom` and as such changes of the default style where not loaded as my style. 
Could you put make 779 and 780 a unique line?

**Comment by Chris00** [comment]
Done.

**Review by jeremy-compostella — APPROVED** [review]

**Pull request merged**